### ISSUE TITLE
demo instance credencials are only available in readme

### ISSUE THREAD

**Issue by ShalokShalom** [gitbucket/gitbucket#1974]
In the current setup is the login information only available in the Github Readme:
Consider to place it somewhere on either the homepage as well, since some people reach the homepage directly.

And those one have no clue or hint, how to login

**Comment by takezoe** [comment]
What do you think about showing default admin id / password with the login form if its password is default. Also, it will push to update the default admin password.

**Comment by ShalokShalom** [comment]
Sounds sane

**Comment by aadrian** [comment]
@takezoe IMHO it's not a good idea. Bots will kill it.

**Comment by RadWolfie** [comment]
I had not seen any software show the default admin / password at login page. If provided, it will be exposed as security exploit when host/customer decide not to change the default password for time being.

The only time default admin / password should ever show is either from readme/setup file or through installation process. (That's the only time I ever know software shows admin / password and best security wise method.)

That's my opinion for host/customer's protection.

**Comment by ShalokShalom** [comment]
This is not correct. In my opinion, this is still an issue. Just because there is no solution yet does not mean automatically that there is no solution. Reopen if you agree that this is still an issue

**Comment by takezoe** [comment]
@ShalokShalom Agree, but I don't have ideas other than I proposed above. Even if you will reopen this issue, I can't do anything.

Of course, if you have any other concrete ideas, you can reopen this issue with them. Or if your idea is showing id / password of the demo site at https://gitbucket.github.io/, you can create a pull request for [this repository](https://github.com/gitbucket/gitbucket.github.io).

 I would like to see your ideas and opinions from the community.

**Comment by McFoggy** [comment]
If we wanted to be really secured during the install process we could generate & log once the admin password if none is provided by some context (env. var for example)

Unfortunately this is not very user friendly as can be a fixed & documented password.

If we limit now the discussion to the demo platform of gitbucket & the intial demand we could enhance the deployment process to set some message with admin password in the information pane.

![image](https://user-images.githubusercontent.com/1119660/41404007-70535922-6fc6-11e8-86a4-40b73f4c7b5c.png)
by adding the following to `gitbucket.conf`
```
information=Gitbucket demo platform, admin login: root/root
```
